### PR TITLE
[Merged by Bors] - refactor(Mathlib/CategoryTheory/Monoidal/Transport): remove `simps` at `transport`

### PR DESCRIPTION
**Pull request by yuma-mizuno**
We get better simp normal forms not by unfolding `transport` but instead putting `simps!` at the definition that uses `transport`. The compiling time is also improved.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by yuma-mizuno** [comment]
!bench

**Comment by yuma-mizuno** [comment]
!bench

File: Mathlib/CategoryTheory/Monoidal/Transport.lean [hunk 1]
```diff
@@ -135,7 +135,7 @@ def fromInduced [MonoidalCategoryStruct D] (F : D ⥤ C) [F.Faithful]
 
 /-- Transport a monoidal structure along an equivalence of (plain) categories.
 -/
-@[simps]
+@[simps (config := {isSimp := false})]
```
Comment (joelriou) [inline Mathlib/CategoryTheory/Monoidal/Transport.lean:138]: It seems that the standard way to do this in mathlib is:
```suggestion
@[simps (config := .lemmasOnly)]
```

**Comment by joelriou** [comment]
Thansk!

bors d+

**Pull request closed without merging**